.gitattributes

```diff
@@ -11,3 +11,9 @@ system/ linguist-vendored
 
 AUTHORS merge=union
 ChangeLog.md merge=union
+
+# line endings
+* text=auto eol=lf
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
```

Comment:
We don't have any files with mixed case extensions like this.  I  think just adding `cmd` and `bat` to the list below should be fine.